src/backend/crossterm.rs

```diff
@@ -127,6 +152,248 @@ where
     }
 }
 
+impl CrosstermBackend<io::Stdout> {
+    /// Creates a new `CrosstermBackend` with `std::io::stdout`.
+    ///
+    /// # Example
+    ///
+    /// ```rust,no_run
+    /// # use ratatui::backend::CrosstermBackend;
+    /// let backend = CrosstermBackend::stdout();
+    /// ```
+    pub fn stdout() -> Self {
+        Self::new(io::stdout())
+    }
+
+    /// Creates a new `CrosstermBackend` with `std::io::stdout` and default settings.
+    ///
+    /// This enables raw mode and switches to the alternate screen. Mouse support is not enabled.
+    ///
+    /// Raw mode and alternate screen are restored when the `CrosstermBackend` is dropped.
+    ///
+    /// # Example
+    ///
+    /// ```rust,no_run
+    /// use ratatui::backend::CrosstermBackend;
+    ///
+    /// let backend = CrosstermBackend::stdout_with_defaults()?;
+    /// # std::io::Result::Ok(())
```

Comment:
One shows the import of itself, one does not. Personally I think both shouldn't show the explicit import of itself as the viewer of the docs found said object already. (There are more cases like this)

Reply:
cd525c335